website-warmup/task.json

```diff
@@ -13,8 +13,9 @@
     "version": {
         "Major": 1,
         "Minor": 2,
```

Comment:
I would suggest updating Minor to 3 and Patch to 0. When adding a non breaking new feature semver suggests updating the Minor version